docs/developer-docs/integrations/ethereum/siwe.md

```diff
@@ -0,0 +1,29 @@
+# Sign in with Ethereum
```

Comment:
```suggestion
# Authenticate to ICP with Ethereum
```

Reply:
The standard is named "Sign-In with Ethereum" or "SIWE" for short. 

https://eips.ethereum.org/EIPS/eip-4361